src/main/webapp/management-console.js

```diff
@@ -5,8 +5,10 @@ angular.module('managementConsole', [
   'gridshore.c3js.chart',
   'ui.router',
   'ui.bootstrap',
-  'LocalStorageModule',
-]).directive('fileModel', ['$parse', function ($parse) {
+  'LocalStorageModule'
+]).constant('CONSTANTS', {
+  'NO_BASE_CONFIGURATION_TEMPLATE': 'NONE - no base template'
```

Comment:
I think it would be best to use the first empty option of that dropdown and make it the default instead of creating a new one for this case. I'd also either leave the text as blank or use angular (hohoho) brackets like "<none>"